src/utils/cognito.ts

```diff
@@ -79,20 +113,35 @@ export const cognitoSignUp = async (params: {
     userPoolId,
     password,
     username,
-    config,
+    clientConfig,
   });
 };
 
+export type DeleteCognitoUserInput = {
```

Comment:
Should this be `CognitoDeleteUserInput` to follow the same naming convention (`CognitoSignUpInput` & `CognitoSignInInput`)?

Also, we probably want to add JSDoc comments for this type

Reply:
good catch. in fact I meant to rename the function :)
I think `deleteCognitoUser()` sounds better than `cognitoDeleteUser()` wdyt?

Reply:
Sounds good, should we also rename the other two functions 👇🏻  to be more consistent?

`cognitoSignUp` --> `signUpCognitoUser`
`cognitoSignIn` --> `signInCognitoUser`

Reply:
I went with 
`signUpCognitoUser` (sign up a user)
 and 
`signInWithCognitoUser` (sign in **with** a user)